### ISSUE TITLE
Replace setup.py with newer package definition standard

### ISSUE THREAD

**Issue by julian-belina** [FZJ-IEK3-VSA/tsam#89]
When installing tsam I get the warning 

" DEPRECATION: Legacy editable install of tsam==2.3.4 from file:///C:/Programming/tsam (setup.py develop) is deprecated. pip 25.0 will enforce this behaviour change. A possible replacement is to add a pyproject.toml or enable --use-pep517, and use setuptools >= 64. If the resulting installation is not behaving as expected, try using --config-settings editable_mode=compat. Please consult the setuptools documentation for more information. Discussion can be found at https://github.com/pypa/pip/issues/11457"

Although a setupy.py file is not deprecated, the use of a pyproject.toml is strongly recommended.

https://packaging.python.org/en/latest/guides/modernize-setup-py-project/

**Comment by l-kotzur** [comment]
@julian-belina Thanks for pointing out. Do you have setup something similar for FINE? Is it easy for you to reproduce here?

**Comment by julian-belina** [comment]
@l-kotzur Yes, I did that for FINE and will do it for TSAM at the next hackathon in about two weeks.
